Build/Newtonsoft.Json.nuspec

```diff
@@ -11,5 +11,34 @@
     <iconUrl>http://www.newtonsoft.com/content/images/nugeticon.png</iconUrl>
     <licenseUrl>https://raw.github.com/JamesNK/Newtonsoft.Json/master/LICENSE.md</licenseUrl>
     <tags>json</tags>
+    <dependencies>      
+      <group targetFramework="net2" />
+      <group targetFramework="sl5" />
+      <group targetFramework="portable-net40+sl5+wp80+win8+wpa81" />
```

Comment:
I will double & triple check this later today, but this is to prevent an error @bradwilson and I saw while adding dotnet support to xUnit yesterday.

It appears that the resolution logic for the dotnet dependencies section is independent from the resolution of the \lib section.

What we saw is that an older platform got the \lib\foo version of the library but the dotnet dependencies section -- unless we added a blank dependency group.

What I want to do later this afternoon is experiment with a number of permutations to see how this all applies. I'm not 100% sure that the blank portable-net4+sl5-... group is needed but I haven't tried what happens yet if I have a PCL of that variety and I try adding a package with the newer dotnet contracts.

The underlying issue is that if the contracts/dependencies in the dotnet section are newer and thus incompatible with the target project, then the package will fail to install using NuGet v3 (v2 ignores the dotnet TFM entirely). The tests involve creating various project types in VS 2015 and adding the package to ensure it works correctly for all.

I'll report back later tonight (NY time) with my findings. I do wish this was better documented, but I'll document my findings.